### PR TITLE
NTBS-2259: Add default no result value for chest xray for all case record

### PR DESCRIPTION
**Pull request by LeoCie**
Add the 'No result' value to ETS records chest xray result.

### CODE REVIEW

**Review by aptemus — APPROVED** [review]
@LeoCie looks good to me

File: source/dbo/Stored Procedures/Power BI Reporting/uspGenerateReportingCaseData.sql [hunk 1]
```diff
@@ -31,6 +31,9 @@ BEGIN TRY
 				WHEN TreatmentOutcome24months IS NOT NULL AND TreatmentOutcome24months != '' AND TreatmentOutcome24months != 'Error: invalid value' THEN TreatmentOutcome24months
 				WHEN TreatmentOutcome12months IS NOT NULL AND TreatmentOutcome12months != '' AND TreatmentOutcome12months != 'Error: invalid value' THEN TreatmentOutcome12months
 			END
+		,ChestXRayResult = CASE
+		WHEN ChestXRayResult IS NULL
+		THEN 'No result' ELSE ChestXRayResult END
```
Comment (JoshCadney) [inline source/dbo/Stored Procedures/Power BI Reporting/uspGenerateReportingCaseData.sql:36]: Why aren't we using `COALESCE` here?

**Review by JoshCadney — CHANGES_REQUESTED** [review]

File: source/dbo/Stored Procedures/Power BI Reporting/uspGenerateReportingCaseData.sql (hunk 1 above)
Comment (LeoCie) [reply source/dbo/Stored Procedures/Power BI Reporting/uspGenerateReportingCaseData.sql]: Friday afternoon!

**Review by JoshCadney — APPROVED** [review]
Looks good :-)

**Pull request merged**